tests/e2e/transitions/sms_workflows.spec.js

```diff
@@ -98,14 +99,26 @@ const expectTasks = (doc, expectations) => {
     .to.have.deep.members(expectations);
 };
 
+const processReportsAndSetings = async (reports, settings) => {
+  const ids = reports.map(report => report._id);
+  await utils.updateSettingsNative(settings);
+  await utils.saveDocsNative(reports);
+  await sentinelUtils.waitForSentinelNative(ids);
+  return utils.getDocsNative(ids);
+};
+
 describe('SMS workflows', () => {
-  beforeAll(() => utils.saveDocs(contacts));
-  afterAll(() => utils.revertDb());
+  beforeAll(async () => { 
+    commonPo.goToPeople();
+    await utils.saveDocsNative(contacts);
+  });
+  
+  afterAll(async () => await utils.revertDbNative());
 
-  afterEach(() => utils.revertDb(contacts.map(c => c._id)));
+  afterEach(async () => await utils.revertDbNative(contacts.map(c => c._id)));
```

Comment:
Does this need the `await` on the `revertDbNative` call? It seems this will return the promise anyway.
```suggestion
  afterEach(async () => utils.revertDbNative(contacts.map(c => c._id)));
```